Gemfile

```diff
@@ -26,6 +26,7 @@ group :development, :test do
   gem "byebug", platforms: %i[mri mingw x64_mingw]
   gem "capybara"
   gem "factory_bot_rails"
+  gem "pry"
```

Comment:
I marginally prefer this to byebug and I think some devs do and some devs don't, so might make sense to have both?

Reply:
Yeah I prefer pry too.. I think we can just have the one gem to cover both though?
We've used this before in the past https://github.com/deivid-rodriguez/pry-byebug

Reply:
I'm not sure it's being given much TLC...last update a year ago and they've not been merging dependabot PRs, including security fixes 🤔